core/src/banking_stage/forwarder.rs

```diff
@@ -160,7 +160,7 @@ impl Forwarder {
 
         self.update_data_budget();
         let packet_vec: Vec<_> = forwardable_packets
-            .filter(|p| !p.meta().forwarded())
+            .filter(|p| !p.meta().forwarded() && p.meta().is_from_staked_node())
```

Comment:
Would this additional logic impact vote forwarding? `Forwarder` is very different from what I worked on, trying to trace it's call stakes. Wondering`ForwardTpuVote` is still a valid path; if so, should TpuVote be excluded?

Reply:
Won't TpuVotes come from staked connections from the other validators?

Reply:
i guess we could just set the bit for everything on the vote port? only staked nodes are submitting votes that won't get dropped anyway

Reply:
yeah, can just do that. Everything in fetch stage is technically from a staked node. Including the forwarded packets.

Reply:
89f8ca427aa2af0651b919bb5c795a6180e431ab

marked tpu votes as from_staked_node. didn't do it for the forwarded packets via udp, since i don't beleive there's a guarantee there.